packages/crud/src/vaadin-crud.js

```diff
@@ -320,7 +319,7 @@ class Crud extends SlotMixin(ControllerMixin(ElementMixin(ThemableMixin(PolymerE
        * @private
        */
       _grid: {
-        type: HTMLElement,
+        type: Object,
```

Comment:
There is no special logic for `type: HTMLElement` in Polymer, so changed to use `type: Object`.